soh/soh/Enhancements/boss-rush/BossRush.cpp

```diff
@@ -522,13 +522,6 @@ void BossRush_OnVanillaBehaviorHandler(GIVanillaBehavior id, bool* should, va_li
             }
             break;
         }
-        // Replace the blue warp transitions with ones that lead back to the chamber of sages
-        case VB_BLUE_WARP_APPLY_ENTRANCE_AND_CUTSCENE: {
-            DoorWarp1* blueWarp = va_arg(args, DoorWarp1*);
-            BossRush_HandleBlueWarp(gPlayState, blueWarp->actor.world.pos.x, blueWarp->actor.world.pos.z);
-            *should = false;
-            break;
-        }
```

Comment:
i couldn't find a `GameInteractor_Should` that uses this

https://github.com/search?q=repo%3AHarbourMasters%2FShipwright%20VB_BLUE_WARP_APPLY_ENTRANCE_AND_CUTSCENE&type=code

Reply:
it was added in https://github.com/HarbourMasters/Shipwright/pull/4255, but i could only find one instance of it in there

Reply:
Potentially related to https://github.com/HarbourMasters/Shipwright/issues/5006